### PR TITLE
Feat: Add `Import-postman-collection` command

### LINKED ISSUE #7: Import Postman Collections as spaces
Add the ability to import postman collections into Explore via the CLI.

This would be limited to API protocols supported by Explore and would not include environments, scripts, or tests.

- [x] - add basic collection imports
- [x] - add nested collection imports
- [x] - add SOAP import ability
- [x] - update ReadMe
- [ ] - prepare release (minor version bump, update ReadMe, add release notes)

### LINKED ISSUE #20: Remove `import-inspector-collections` command as Inspector has been sunset
remove code and update ReadMe

### PR DESCRIPTION
**Pull request by frankkilcommins**
Add ability to import postman collection into explore. This PR also removes the `import-inspector-collection` command as Swagger Inspector has been retired.

Fixes #7
Fixes #20

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,6 +25,8 @@ Simple utility CLI for importing data into SwaggerHub Explore.
 >  `export-spaces`                 Export SwaggerHub Explore spaces to filesystem
 >
 >  `import-spaces`                 Import SwaggerHub Explore spaces from a file
+>
+>  `import-postman-collection`     Import Postman Collection from a file into SwaggerHub Explore
```
Comment (YOU54F) [inline README.md:29]: worth adding the version here? `v2.1 Postman Collection`

**Review by YOU54F — APPROVED** [review]
Tested locally, with a v2.0 and v2.1 collection exported from Postman. v2.1 worked fine, v2.0 returns error 

```
An error occurred: The JSON value could not be converted to Url. Path: $.item[0].request.url | LineNumber: 13 | BytePositionInLine: 78.
```

Would be nice to gracefully handle and suggest the user converts to v2.1 prior to upload.

Noted that it is mentioned in the readme re compliance with v2.1 only, worth probably popping that into the tool `--help` text with regards to a `v2.1 Postman Collection`

As it stands it is good to go, and you can either address comments later

File: README.md (hunk 1 above)
Comment (frankkilcommins) [reply README.md:29]: added

**Comment by YOU54F** [comment]
Appreciate the extra validation Frank!

**Pull request merged**